WalletWasabi/Tor/Socks5/TorTcpConnection.cs

```diff
@@ -25,7 +25,13 @@ public class TorTcpConnection : IDisposable
 	public TorTcpConnection(TcpClient tcpClient, Stream transportStream, ICircuit circuit, bool allowRecycling)
 	{
 		long id = Interlocked.Increment(ref LastId);
-		Name = $"TC#{id:0000}#{circuit.Name[0..10]}";
+		string prefix = circuit switch
+		{
+			DefaultCircuit _ => "DC",
+			PersonCircuit _ => "PC",
+			_ => "UC" // Unknown circuit type.
+		};
+		Name = $"{prefix}#{id:0000}#{circuit.Name[0..10]}";
```

Comment:
This helps a lot in debugging because one can immediately see whether an HTTP request is sent over default circuit or person circuit.